contentcuration/contentcuration/urls.py

```diff
@@ -179,6 +179,10 @@ def get_redirect_url(self, *args, **kwargs):
     url(r'^api/deferred_user_data/$', registration_views.deferred_user_data, name="deferred_user_data"),
     url(r'^settings/$', settings_views.settings, name='settings'),
     url(r'^administration/', admin_views.administration, name='administration'),
-    url(r'^admin/', include(admin.site.urls)),
```

Comment:
Can just remove from here, and add here instead: https://github.com/learningequality/studio/blob/hotfixes/contentcuration/contentcuration/dev_urls.py

Reply:
Done